packages/gatsby-theme-newrelic/src/components/Popover/PopoverButton.js

```diff
@@ -0,0 +1,142 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import { Icon, useTessen } from '@newrelic/gatsby-theme-newrelic';
```

Comment:
i think we can just directly import these, since we're in the package, right?

Reply:
yep, updated!